packages/node/src/plugins/UniversalFederationPlugin.ts

```diff
@@ -45,14 +46,45 @@ class UniversalFederationPlugin {
     this._options = options || ({} as NodeFederationOptions);
     this.context = context || ({} as NodeFederationContext);
     this.name = 'ModuleFederationPlugin';
+    if (this._options.useRuntimePlugin && this._options.isServer) {
```

Comment:
@2heal1 is this the right way to set these options? 
I find if i do not set these options here, federationRuntimePlugin will get wrong options from runtimePlugin as userConfig does now have this value set.